### PR TITLE
Update with installation section only

### PR DESCRIPTION
**Pull request by arlene-perez**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,35 +10,38 @@ Creating what we call [starter issues](http://hood.ie/blog/starter-issues.html)
...
+Say I’m a Hoodie contributor and find a typo somewhere. Instead of fixing the issue directly in the master branch or creating a pull request which is time-consuming, I can simply create a new branch that is called something like **first-timers-only-typo-in-title.** GitHub will then notify the "first-timers-only issue bot" about the new branch using Webhooks. The bot is listening to any new branch starting with **first-timers-only** and it will create a new issue on your repo. It is currently setup to the Hoodie repo with a template and assigns the first-timers-only and up-for-grabs labels but can be modified by the developer. The commit body can be used to add some context information and if left empty, the 🤔 **What you will need to know** section of the issue will simply say "Nothing :)".
```
Comment (gr2m) [inline README.md]: I would replace `"first-timers-only issue bot"` => `First Timers Bot`

File: README.md [hunk 2]
```diff
@@ -10,35 +10,38 @@ Creating what we call [starter issues](http://hood.ie/blog/starter-issues.html)
...
+        <td><img src="/assets/Edit.png?raw=true"></td>
+    </tr>
+    <tr>
+        <td>3) Make the change and write your commit message under <b>Commit changes</b>.  Make sure to check <i>Create a new branch</i> at the bottom and begin the title of the branch with <b>first-timers-only</b> for example.</td>
+        <td><img src="/assets/Committing-Branch.png?raw=true"></td>
+    </tr>
+    <tr>
+        <td>4) Click on the <b>issues</b> tab and notice your issue was created with your change and commit message. The contributor would then follow the steps on the issue message.</td>
+        <td><img src="/assets/Issue-Generated.png?raw=true"</td>
+</tr>
```
Comment (gr2m) [inline README.md]: nitpick: indentation :)

File: README.md [hunk 3]
```diff
@@ -10,35 +10,38 @@ Creating what we call [starter issues](http://hood.ie/blog/starter-issues.html)
...
+
+### 😮 🙌 👀 🎉 Use Our Bot!
+
+<table>
+    <tr>
+        <th>Steps</th>
+        <th>Example</th>
+    </tr>
+    <tr>
+        <td>1) <a href="https://github.com/apps/first-timers-bot">Install App</a> on a repo of your choice</td>
```
Comment (gr2m) [inline README.md]: and an empty `<td></td>`. Or add a screenshot of the install page?

image

File: README.md [hunk 4]
```diff
@@ -10,35 +10,38 @@ Creating what we call [starter issues](http://hood.ie/blog/starter-issues.html)
...
+Say I’m a Hoodie contributor and find a typo somewhere. Instead of fixing the issue directly in the master branch or creating a pull request which is time-consuming, I can simply create a new branch that is called something like **first-timers-only-typo-in-title.** GitHub will then notify the "first-timers-only issue bot" about the new branch using Webhooks. The bot is listening to any new branch starting with **first-timers-only** and it will create a new issue on your repo. It is currently setup to the Hoodie repo with a template and assigns the first-timers-only and up-for-grabs labels but can be modified by the developer. The commit body can be used to add some context information and if left empty, the 🤔 **What you will need to know** section of the issue will simply say "Nothing :)".
+
+
+### 😮 🙌 👀 🎉 Use Our Bot!
```
Comment (gr2m) [inline README.md]: I would remove the spaces between the emojis

File: README.md [hunk 5]
```diff
@@ -10,35 +10,38 @@ Creating what we call [starter issues](http://hood.ie/blog/starter-issues.html)
...
+        <td>3) Make the change and write your commit message under <b>Commit changes</b>.  Make sure to check <i>Create a new branch</i> at the bottom and begin the title of the branch with <b>first-timers-only</b> for example.</td>
+        <td><img src="/assets/Committing-Branch.png?raw=true"></td>
+    </tr>
+    <tr>
+        <td>4) Click on the <b>issues</b> tab and notice your issue was created with your change and commit message. The contributor would then follow the steps on the issue message.</td>
+        <td><img src="/assets/Issue-Generated.png?raw=true"</td>
+</tr>
+</table>
+
+### 😱 🙌 😎Result
```
Comment (gr2m) [inline README.md]: remove spaces between emojis, add space before `Result`

File: README.md (hunk 2 above)
Comment (agonzalez0515) [reply README.md]: The branch to be picked up just needs to start with "first-timers-"
Check out line 7 of server.js.

**Comment by gr2m** [comment]
when you merge https://github.com/hoodiehq/first-timers-only-bot/pull/83 (see my change requests first) and then rebase this PR, travis will pass :)

**Pull request merged**